vscode-dotnet-runtime-library/distro-data/distro-support.json

```diff
@@ -164,37 +179,153 @@
 				]
 			},
 			{
-				"version": "22.04",
-				"dotnet": [{
-						"version": "6.0",
-						"sdk": "dotnet-sdk-6.0",
-						"runtime": "dotnet-runtime-6.0",
-						"aspnetcore": "aspnetcore-runtime-6.0"
-					},
-					{
-						"version": "7.0",
-						"sdk": "dotnet-sdk-7.0",
-						"runtime": "dotnet-runtime-7.0",
-						"aspnetcore": "aspnetcore-runtime-7.0"
-					}
+				"version": "22.04"
+			},
+			{
+				"version": "23.04"
+			}
+		]
+	},
+	"Red Hat Enterprise Linux": {
+		"installCommand": [
+			{
+				"runUnderSudo": true,
+				"commandRoot": "dnf",
+				"commandParts": [
+					"install",
+					"-y",
+					"{packageName}"
+				]
+			}
+		],
+		"uninstallCommand": [
+			{
+				"runUnderSudo": true,
+				"commandRoot": "dnf",
+				"commandParts": [
+					"remove",
+					"-y",
+					"{packageName}"
+				]
+			}
+		],
+		"updateCommand": [
+			{
+				"runUnderSudo": true,
+				"commandRoot": "dnf",
+				"commandParts": [
+					"update",
+					"-y",
+					"{packageName}"
+				]
+			}
+		],
+		"searchCommand": [
+			{
+				"runUnderSudo": false,
+				"commandRoot": "yum",
+				"commandParts": [
+					"search",
+					"{0}",
+					"-q"
+				]
+			}
+		],
+		"isInstalledCommand": [
+			{
+				"runUnderSudo": true,
+				"commandRoot": "dnf",
+				"commandParts": [
+					"list",
+					"--installed",
+					"{packageName}",
+					"-q"
+				]
+			}
+		],
+		"packageLookupCommand": [
+			{
+				"runUnderSudo": false,
+				"commandRoot": "yum",
+				"commandParts": [
+					"list",
+					"install",
+					"{packageName}",
+					"-q"
+				]
+			}
+		],
+		"expectedDistroFeedInstallDirectory": "/usr/lib64/dotnet/sdk",
```

Comment:
```suggestion
		"expectedDistroFeedInstallDirectory": "/usr/lib64/dotnet/dotnet",
```

Reply:
We just made a modification to the API so it returns the path to the dotnet object file to execute, so this should also be changed here